Source/WebCore/platform/graphics/GeometryUtilities.h

```diff
@@ -35,8 +35,8 @@ namespace WebCore {
 
 class FloatQuad;
 
-float euclidianDistance(const FloatSize&);
-float euclidianDistance(const FloatPoint&, const FloatPoint&);
+WEBCORE_EXPORT float euclidianDistance(const FloatSize&);
```

Comment:
You don't need to export this one, right? Since you're only calling the latter.

Reply:
Yes good point! Removed it